### DISCUSSION TITLE
Custom Block Hardness etc..?

### DISCUSSION THREAD

**Discussion by derverdox** [Minestom/Minestom#1192] [Help]
So every Block has its own registry. Can I create a variation of a Block with different hardness from the Block class?

like Block.Stone.withHardness(float);?

**Comment by Flopgop** [comment]
You have to send individual break animation packets yourself in order to modify the hardness of the block.

**Reply by derverdox** [reply]
> You have to send individual break animation packets yourself in order to modify the hardness of the block.

And there is no code yet to create blocks with its own data? 

Thanks!

**Reply by Flopgop** [reply]
you have to add blocks to the blocks.json in your resources, I'm not sure of the exact format though.
